silverback/_click_ext.py

```diff
@@ -242,3 +244,16 @@ def get_cluster_client(ctx: click.Context, *args, **kwargs):
         return ctx.invoke(f, *args, **kwargs)
 
     return update_wrapper(get_cluster_client, f)
+
+
+def path_callback(ctx: click.Context, param: click.Parameter, value):
```

Comment:
suggestion for clarity
```suggestion
def bot_path_callback(ctx: click.Context, param: click.Parameter, value):
```